spec/yelp/endpoint/business_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:api_keys) { real_api_keys }
   let(:business) { 'yelp-san-francisco' }
   let(:client) { Yelp::Client.new(api_keys) }
+  let(:locale) { Hash[lang: 'fr'] }
```

Comment:
`fr` is a nice language, but it's not a locale --- that'd be `fr_FR`, `fr_CA`, or similar.